custom_components/places/switch.py

```
@@ -35,18 +38,38 @@ class PlacesShowLastUpdatedSwitch(PlacesEntity, SwitchEntity):
     @property
     def is_on(self) -> bool:
-        """Return whether the suffix is enabled."""
+        """Return whether the suffix is enabled.
+
+        Returns:
+            bool:
+                The value.
+        """
         return bool(self.coordinator.get_attr(CONF_SHOW_TIME))
 
     async def async_turn_on(self, **kwargs: object) -> None:
```

Ruff (0.16.1)

[warning] Unused method argument: `kwargs`

(ARG002)